src/aioswitcher/device/tools.py

```diff
@@ -211,3 +213,32 @@ def validate_token(username: str, token: str) -> bool:
     else:
         logger.debug("request failed with status code: %s", response.status_code)
     return is_token_valid
+
+
+def get_shutter_index(device_type: DeviceType, device_num: int) -> int:
+    """Return the correct shutter index.
+
+    Used in retriving the shutter position/direction from the packet
+    (based of device type and device number).
+    """
+    if device_type == DeviceType.RUNNER or device_type == DeviceType.RUNNER_MINI:
```

Comment:
I can only assume this is a preparation for future devices not `S11` but for now it doesn't make sense to check `device_num` in each conditon.
```python
if device_num != 0:
    raise ValueError("Invalid device number")

if device_type in (DeviceType.RUNNER, DeviceType.RUNNER_MINI):
   return 0
elif device_type == DeviceType.RUNNER_S11:
  return 2

raise ValueError("only shutters are allowed")
```

Reply:
Changes to the code suggested.